validator/sawtooth_validator/journal/chain.py

```
@@ -94,8 +94,14 @@ def _is_block_complete(self, blkw):
         return True
 
     def _verify_block_signature(self, blkw):
-        return signing.verify(blkw.block.header, blkw.block.header_signature,
-                              blkw.header.signer_pubkey)
+        try:
+            return signing.verify(
+                blkw.block.header,
+                blkw.block.header_signature,
+                blkw.header.signer_pubkey)
+        # pylint: disable=broad-except
+        except Exception:
```

Why is this bare except needed? Please add a comment stating why.